alica_ros_turtlesim/alica/etc/turtle1/Local.conf

```diff
@@ -1,6 +1,4 @@
 [Local]
-	[turtle1]
-		ID = 1 
-		DefaultRole = Turtle
-	[!turtle1]
+	ID = 1 
```

Comment:
Is there no name associated with the agents anymore?

Reply:
> ?

Agents still have a name, it's just not part of conf file.
AlicaContext::setRobotName() api can be used for same. Default is the real host name.